src/components/Storage/Canvas/BucketPicker/BucketPicker.test.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2019 Google LLC
```

Comment:
New files should have `2021` as the copyright year. (I see a few 2019s and 2020s throughout new files on this PR and you may want to do a global find and replace for the changed files)